features/cart/shopping_cart/adding_product_with_variants_with_discounted_catalog_price_to_cart.feature

```diff
@@ -0,0 +1,18 @@
+@shopping_cart
+Feature: Adding a product with selected variant with discounted catalog price to the cart
+    In order to select products with proper price
+    As a Visitor
+    I want to be able to add products with selected variants  to cart
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "T-Shirt"
+        And this product has "PHP T-Shirt" variant priced at "$20"
```

Comment:
To make it essentially different from the previous feature, I would add another variant here - to point out the discount is applied only on one of them

Reply:
I have just added the second product and variant (not discounted) and add it to checks 🎉